codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -86,30 +86,43 @@ As one dynamic contract register catches up to the fetched blocknumebr of the ne
 merge itself into the next register and combine queries/addresses and queues until fully caught
 up to the root. 
 */
-type fetchStateData = {
+type registerData = {
   latestFetchedBlock: blockNumberAndTimestamp,
   contractAddressMapping: ContractAddressingMap.mapping,
   //Events ordered from latest to earliest
   fetchedEventQueue: array<Types.eventBatchQueueItem>,
   //Used to prune dynamic contract registrations in the event
   //of a rollback.
   dynamicContracts: DynamicContractsMap.t,
-  isFetchingAtHead: bool,
   firstEventBlockNumber: option<int>,
 }
 
-type rec t = {
-  registerType: register,
-  ...fetchStateData,
+type rec register = {
+  registerType: registerType,
+  ...registerData,
+}
```

Comment:
Some renaming going on over here. What was `FetchState.t` is now simply a "register",  `FetchState.t` is a record that contains a "base register" and some additional state.